VMEncryption/main/common_parameters.json

```diff
@@ -1,5 +1,5 @@
 {
-    "extension_version": "1.4.0.9",
+    "extension_version": "1.4.0.17",
```

Comment:
ToDo: update to next version

Next version is 1.4.0.11 I believe, I think the last one was not merged?

Reply:
{
    "name": "AzureDiskEncryptionForLinux",
    "publisher": "Microsoft.Azure.Security",
    "version": "1.4.0.10"
  },
  
  1.4.0.10 is released